randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -419,6 +436,8 @@ Extra - asset_id: 11
           Any of the following:
               Morph Ball Bombs or Power Bombs ≥ 4
               Power Bombs and Screw Attack
```

Comment:
These can get a little refactored IMO
- Lay Bombs or (Lay PB && PB >= 4)
- Lay PB && Screw # Destroy the initial blocks with PB, and the rest with screw

Same goes for going back

Reply:
Morph is required for all of these,, you want canlaybombs instead of bombs even though morph is already required?

Reply:
Yes, it's nicer to read that way

Reply:
Is this better? 
![image](https://github.com/randovania/randovania/assets/46821362/5d122f29-127f-4cd5-890b-f17e2fb577c1)

Reply:
use templates for the other 3 too please

Reply:
updated with templates